addons/website/views/snippets/s_timeline.xml

```diff
@@ -2,51 +2,58 @@
 <odoo>
 
 <template name="Timeline" id="s_timeline">
-    <section class="s_timeline pt24 pb48">
-        <div class="container s_timeline_line">
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2019</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <h5 class="card-title">Your title</h5>
-                            <p class="card-text">A timeline is a graphical representation on which important events are marked.</p>
-                        </div>
-                    </div>
-                    <i class="fa fa-1x fa-child bg-o-color-2 rounded-circle s_timeline_icon"/>
+    <section class="s_timeline pt48 pb48" data-vcss="001">
+        <div class="o_container_small">
+            <div class="row">
+                <div class="col-12 pb24 text-center" data-name="Heading">
+                    <h2 class="h3-fs">Latest news</h2>
+                    <p class="lead">Highlight your history, showcase growth and key milestones.</p>
                 </div>
-                <div class="s_timeline_content"/>
             </div>
-            <div class="s_timeline_row d-block d-md-flex flex-row" data-name="Row">
-                <div class="s_timeline_date"><b>2018</b></div>
-                <div class="s_timeline_content d-flex">
-                    <div class="s_timeline_card s_card card bg-white w-100" data-name="Card" data-snippet="s_card">
-                        <div class="card-body">
-                            <p class="card-text">You can edit, duplicate...</p>
+            <div class="position-relative">
+                <div class="s_timeline_line position-absolute top-0 bottom-0 d-block border-start"/>
```

Comment:
Given that that it is at the same level as the `s_timeline_row` elements, the line is considered as a child too and so when we click on the topmost "milestone", the overlay arrow tells that we can still move it upwards, even though it is the first one => when we look in the DOM after clicking on the arrow, we see that it in fact exchanged places with the line :open_mouth: And when we look at the snippet, the line now goes over the dot :grimacing: 

![image](https://github.com/user-attachments/assets/e26a834a-2818-4692-b973-dfa83dbc2d7f)
![image](https://github.com/user-attachments/assets/049d4737-48dd-4df0-840c-c17935afc575)

 => Maybe you should find another place to add the `s_timeline` element :slightly_smiling_face: